common/remote/hooks/libarchive/archive.cpp

```diff
@@ -385,6 +385,13 @@ class ArchiveFile : implements IFile, public CInterface
             return createArchiveDirectoryIterator(dirName, mask, sub, includeDirs);
         }
     }
+    IDirectoryIterator *directoryFilesSorted(SortDirectoryMode mode, bool rev, const char *mask=NULL, bool sub=false, bool includedirs=false)
```

Comment:
As above, should be consistent with 'virtual' and don't need default params.
And would benefit from 'override', but should add to other methods to be consistent.